my_portfolio.html

```diff
@@ -1,13 +1,63 @@
-﻿<!DOCTYPE html>
-
+<!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>My Portfolio</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
     <link rel="stylesheet" href="style.css" />
 </head>
 <body>
-    <h1>Hello world</h1>
+    <div id="header-img"></div>
+    <div id="header-img-desktop"></div>
+    <div id="general-container">
+        <header id="nav-bar-ctn">
+            <div id="nav-bar">
+                <a>
+                    <p>Portfolio</p>
+                </a>
+                <a>
+                    <p>About</p>
+                </a>
+                <a>
+                    <p>Contact</p>
+                </a>
+            </div>
+        </header>
+        <nav id="toolbar">
+            <div id="menu-button">
+                <object data="icons/icon-menu.svg"></object>
+            </div>
+        </nav>
+        <section id="headline">
+            <h1 id="primary-text">
+                <p id="title-1">Hey there. I'm Gumaro.</p>
+                <p id="title-2">I'm a software developer</p>
+            </h1>
+            <div id="support-text">
+                <p>
+                    I can help you build a product , feature or website Look through some of my work and experience! If you
+                    like what you see and have a project you need coded,
+                    don’t hesitate to contact me.
+                </p>
+            </div>
+            <div id="social-media">
+                <ul id="soc-med-list">
+                    <li><img src="icons/twitter.svg" /></li>
+                    <li><img src="icons/linkedin.svg" /></li>
+                    <li><img src="icons/medium.svg" /></li>
+                    <li><img src="icons/github.svg" /></li>
+                    <li><img src="icons/angellist.svg" /></li>
```

Comment:
Great job! As per the previous reviewer's request, you have created a fantastic social icon list. However, there are a few additional changes that still need to be made:

Please make sure to wrap each "ima" element inside an "<a>" tag and include the "href" attribute. This is important for proper functionality and linking the icons to their respective destinations using `href ` attribute.